postgres/db-init/init.sql

```diff
@@ -1,27 +1,26 @@
 CREATE TABLE emails2send (
-    uid VARCHAR(36) UNIQUE NOT NULL,
-    name TEXT NOT NULL,
-    phone TEXT NOT NULL,
-    email TEXT NOT NULL,
-    content TEXT NOT NULL,
-    created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6)
+    "uid" VARCHAR(36) UNIQUE NOT NULL,
+    "to" TEXT NOT NULL,
+    "subject" TEXT NOT NULL,
+    "content" TEXT NOT NULL,
+    "created_at" TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6)
 );
 
 CREATE TABLE library (
-    uid VARCHAR(36) UNIQUE NOT NULL,
-    name TEXT NOT NULL,
-    author TEXT NOT NULL,
-    email TEXT,
-    description TEXT NOT NULL,
-    micheline TEXT NOT NULL,
-    michelson TEXT NOT NULL,
-    created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
-    status SMALLINT NOT NULL
+    "uid" VARCHAR(36) UNIQUE NOT NULL,
```

Comment:
is there any particular reason to use " here?

Reply:
no, will delete. 
FYI: For a moment when I was working on that I added column `from` and thats why I wrapped column names with quotes

Reply:
Update: I need to use quotes, when I deleted them I got:
```
/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
psql:/docker-entrypoint-initdb.d/init.sql:7: ERROR:  syntax error at or near "to"
LINE 3:     to TEXT NOT NULL,
            ^
ERROR:  syntax error at or near "to" at character 69
STATEMENT:  CREATE TABLE emails2send (
	    uid VARCHAR(36) UNIQUE NOT NULL,
	    to TEXT NOT NULL,
	    subject TEXT NOT NULL,
	    content TEXT NOT NULL,
	    created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6)
	);
```

when creating a table. Apparently `to` a special keyword.

So I will leave all column names in quotes, is that OK with you ?